Jenkinsfile.internal

```diff
@@ -44,6 +44,7 @@ pipeline {
     }
 
     parameters {
+        string(name: 'BRANCH_NAME', defaultValue: 'main', description: 'The specified branch name to use for testing.')
```

Comment:
Too generic name. Somehow indicate, that its the branch for the containertest data